projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -24,7 +24,7 @@ import { PromptItemProps, PromptTypeProp, buildPromptForExtensions } from '../..
 export const withAIAssistant = createHigherOrderComponent(
 	BlockEdit => props => {
 		const { clientId } = props;
-		const [ prompt, setPrompt ] = useState< Array< PromptItemProps > >( [] );
+		const [ storedPrompt, storePromptInLocalState ] = useState< Array< PromptItemProps > >( [] );
```

Comment:
```suggestion
		const [ storedPrompt, setStoredPrompt ] = useState< Array< PromptItemProps > >( [] );
```
To keep with convention